org.torproject.torbrowser.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.torproject.torbrowser
+
+runtime: org.gnome.Platform
+runtime-version: "3.32"
+sdk: org.gnome.Sdk
+
+command: launch.sh
+
+finish-args:
+  - --socket=x11
+  # - --share=ipc
+  - --share=network
+
+modules:
+  - name: torbrowser
+    buildsystem: simple
+    build-commands:
+        - install -D Browser/browser/chrome/icons/default/default64.png   /app/share/icons/hicolor/64x64/apps/org.torproject.torbrowser.png
+        #- install -D Browser/start-tor-browser.desktop /app/share/applications/org.torproject.browser.desktop
+        - sed -i s,Icon=torbrowser,Icon=org.torproject.torbrowser,  torbrowser.desktop
+        - sed -i "s,Exec=torbrowser-launcher %u,Exec=/app/bin/launch.sh,"  torbrowser.desktop
+        - install -D torbrowser.appdata.xml  /app/share/metainfo/org.torproject.torbrowser.appdata.xml
+        - install -D torbrowser.desktop  /app/share/applications/org.torproject.torbrowser.desktop
+        - mv Browser /app
+        - install -D -m a=rx launch.sh   /app/bin/launch.sh
+    
+    cleanup:
+      - /Browser/TorBrowser/Docs/
+    sources:
+        - type: file
+          url: https://raw.githubusercontent.com/micahflee/torbrowser-launcher/develop/share/metainfo/torbrowser.appdata.xml
+          sha256: 4aa47248ce6ea3b54fe349bfa03169dc72cceb2f206d517cbf37eeab929d68d0
+
+        - type: file
+          url: https://raw.githubusercontent.com/micahflee/torbrowser-launcher/develop/share/applications/torbrowser.desktop
+          sha256: edc4a08adcfb78468a34bd46ef8f0de0ef5fdb141bd438709e755dc8b2a3eee6
+
+        - type: script
+          commands:
+            # Yes. This is dirty. The tarball contains a Firefox profile directory which Firefox expects to be able to write...
+            # The tmp directory is, at least when following this manifest and not overwriting the settings, a tmpfs, so we can (or have to) copy over everytime we launch.
+            - cp -ar --reflink=auto /app/Browser /tmp/
+            - cd /tmp/
+            - cd Browser
+            - ./start-tor-browser
+          dest-filename: launch.sh
+
+        - type: archive
+          url: https://www.torproject.org/dist/torbrowser/8.5.3/tor-browser-linux64-8.5.3_en-US.tar.xz
```

Comment:
Add all supported languages

Reply:
how would that work?
I've manually diffed the en-US with another version and the only difference I found was the relevant language pack extension weighing a few kilobytes. Downloading the whole Firefox only to get hold of the language pack seems very wasteful. 
I think it's similar to this problem: https://github.com/flathub/org.mozilla.Thunderbird/issues/3